bugbug/models/bugtype.py

```diff
@@ -170,20 +106,23 @@ def get_labels(self) -> tuple[dict[int, np.ndarray], list[str]]:
         bug_map = {bug["id"]: bug for bug in bugzilla.get_bugs()}
 
         for bug_data in bug_map.values():
-            target = np.zeros(len(TYPE_LIST))
-            for type_ in bug_to_types(bug_data, bug_map):
-                target[TYPE_LIST.index(type_)] = 1
+            target = np.zeros(len(self.label_extractors))
+            for i, is_type in enumerate(self.label_extractors):
+                if is_type(bug_data, bug_map):
+                    target[i] = 1
 
             classes[int(bug_data["id"])] = target
 
-        for type_ in TYPE_LIST:
+        type_list = [extractor.type_name for extractor in self.label_extractors]
```

Comment:
More descriptive and consistent with `bug_type` that we use in the loop:
```suggestion
        bug_types = [extractor.type_name for extractor in self.label_extractors]
```